ioctl/newcmd/account/accountdelete.go

```diff
@@ -102,17 +100,15 @@ func NewAccountDelete(c ioctl.Client) *cobra.Command {
 			ks := c.NewKeyStore(config.ReadConfig.Wallet, keystore.StandardScryptN, keystore.StandardScryptP)
 			for _, v := range ks.Accounts() {
 				if bytes.Equal(account.Bytes(), v.Address.Bytes()) {
-					var confirm string
 					message := output.ConfirmationMessage{Info: infoWarn, Options: []string{"yes"}}
 					fmt.Println(message.String())
-					fmt.Scanf("%s", &confirm)
-					if !strings.EqualFold(confirm, "yes") {
+					if !c.AskToConfirm() {
 						output.PrintResult("quit")
 						return nil
 					}
 
 					if err := os.Remove(v.URL.Path); err != nil {
-						return output.NewError(output.WriteFileError, failToRemoveKeystoreFile, err)
+						return output.NewError(output.ReadFileError, failToRemoveKeystoreFile, err)
```

Comment:
if cannot remove file, it should be `WriteFileError`

Reply:
ok

Reply:
address this comment before merging